index.html

```diff
@@ -53,5 +46,124 @@ <h3>I'am a software developer</h3>
+            <div class="project-image">
+                <div class="single-title">Professional Art<br> Printing Data</div>
+                <div class="single-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+                <div class="single-language">
+                    <p>html</p>
+                    <p>bootstrap</p>
+                    <p>Ruby</p>
+                </div>
+                
+            </div>
+            <div class="view-project">
+                <p>See Project</p>
+            </div>
+        </div>
+        <div class="single-project">
+            <div class="project-image">
+                <div class="single-title">Professional Art<br> Printing Data</div>
+                <div class="single-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+                <div class="single-language">
+                    <p>html</p>
+                    <p>bootstrap</p>
+                    <p>Ruby</p>
+                </div>
+                
+            </div>
+            <div class="view-project">
+                <p>See Project</p>
+            </div>
+        </div>
+        <div class="single-project">
+            <div class="project-image">
+                <div class="single-title">Professional Art<br> Printing Data</div>
+                <div class="single-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+                <div class="single-language">
+                    <p>html</p>
+                    <p>bootstrap</p>
+                    <p>Ruby</p>
+                </div>
+                
+            </div>
+            <div class="view-project">
+                <p>See Project</p>
+            </div>
+        </div>
+        <div class="single-project">
+            <div class="project-image">
+                <div class="single-title">Professional Art<br> Printing Data</div>
+                <div class="single-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+                <div class="single-language">
+                    <p>html</p>
+                    <p>bootstrap</p>
+                    <p>Ruby</p>
+                </div>
+                
+            </div>
+            <div class="view-project">
+                <p>See Project</p>
+            </div>
+        </div>
+        
+    </main>
+    <section class="about-me">
+        <h3>About Me</h3>
+        <p id="about-me-title">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button>Get My Resume</button>
+    </section>
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).So Kindly add the technology categories ("Languages", "Frameworks" and "Skills") Just Like the Figma design.
- [ ] And Also Kindly I recommend to make the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li ` and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li` tag.
- The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/213761384-619876ec-18e0-48a1-8cd7-1fe6c1666dc5.PNG)